mpf/mpfconfig.yaml

```diff
@@ -245,6 +246,10 @@ fadecandy:
     keyframe_interpolation: True
     dithering: True
 
+pkone:
+    port: com3
```

Comment:
I guess we should remove this block. Config validation already contains the baud rate and the port will be different for most people.